docs/docs/contributing/intellij-idea.md

```diff
@@ -20,40 +20,51 @@ its installed on your local machine). Otherwise, specify it by pressing *New*.
 
 ![](../../images/idea/idea-sdk.png "Select the JDK")
 
-On the final window we must select which modules we can import. Here, we are presented with the full list of SBT projects
+In some versions of IDEA, we must select which modules we can import. Here, we are presented with the full list of SBT projects
 that are defined in Dotty. You can either select all (expect performance degradation by IDEA if you select all) or
 select only the `dotty` module. In order to do that, unselect all modules and select on `dotty`. IDEA, then, automatically
 selects all the necessary dependencies and you press OK.
 
 ![](../../images/idea/idea-sbt.png "Select modules to import")
 
+In other versions of IDEA, no choice will be given. After the project loads, right click on `dotty` in the project explorer
+and select `Load/Unload Modules...`. Then select only `dotty`, accept when IDEA asks to add its dependencies and press OK.
+
 Running/Debugging
 -------
 
-To run the compiler you can do it either as an sbt command or via debugging the compiler.
-For the first option you can fire up sbt from the `Terminal` window of IDEA or you can do it externally.
+To run the compiler you can do it either as an sbt command or a shell script.
+For the first option you can fire up sbt from the `Terminal` window of IDEA or you from an external terminal.
 For example, to run a test you can write with or without flags:
 
 ```shell
-$ dotc tests/pos/Arrays.scala
+$ sbt
+> dotc tests/pos/Arrays.scala
+```
+For the second option, the compiler can be run using a script in the `bin` directory:
+
+```shell
+$ ./bin/dotc tests/pos/Arrays.scala
 ```
 
-If you are interested in debugging the compiler you can enable the necessary agent on the JVM.
-This is done in two steps. For the first step you need to pass the 
-necessary flag to the running VM. For convenience, this is already in comments on the `Build.scala` file under the
+If you are interested in debugging the compiler, you can use a remote debugging configuration.
```

Comment:
Actually this step is automated now. We run `dist/pack` from sbt and then we pass the `-debug` flag when compiling with `dotc`. I think this is a great opportunity to write this small improvement down.

Reply:
Thanks, that way is more convenient.

I noticed it does not work from within sbt, although that is not important. It fails with a `bad option '-debug' was ignored` error. Also, perhaps it is by design, but neither the sbt `dotc` nor the script `dotc` list `-debug` in the options under the `-help` command.

Reply:
I just found that the `dist/pack` command was not necessary for debug to work. Do you have a guess as to what I am doing differently? I am using the most recent commit of the master branch.

Reply:
> I noticed it does not work from within sbt

Indeed, `debug` does not work  within sbt. It might be tricky to make it work though

> I just found that the dist/pack command was not necessary for debug to work.

Yeh, `dist/pack` is not necessary. If you run `bin/dotc`, the script will run `dist/pack` for you if needed.

> Also, perhaps it is by design, but neither the sbt dotc nor the script dotc list -debug in the options under the -help command.

I don't think we should list the `debug` option with the `help` command. This an internal option meant to debug the compiler.